src/hm-admin/services/config/config.ts

```diff
@@ -0,0 +1,18 @@
+import {Injectable} from '@angular/core';
+
+@Injectable()
+export class ConfigService {
+  api: any = {};
+
+  /**
+   * @param _config
+   * @private
+   */
+  _hookMeIfYouCan(_config: any): void {
```

Comment:
since the styleguide is out, we have to rename all private function and variable to remove `_` prefix